src/Sentry/SentryHttpMessageHandler.cs

```diff
@@ -63,16 +64,27 @@ public SentryHttpMessageHandler()
 
             // Start a span that tracks this request
             // (may be null if transaction is not set on the scope)
+            var requestMethod = request.Method.Method.ToUpperInvariant();
             var span = _hub.GetSpan()?.StartChild(
                 "http.client",
                 // e.g. "GET https://example.com"
-                $"{request.Method.Method.ToUpperInvariant()} {request.RequestUri}"
+                $"{requestMethod} {request.RequestUri}"
             );
 
             try
             {
                 var response = await base.SendAsync(request, cancellationToken).ConfigureAwait(false);
 
+                if (request.RequestUri?.AbsoluteUri is { } uri)
+                {
+                    var breadcrumbData = new Dictionary<string, string>
+                    {
+                        { "url", uri },
+                        { "method", requestMethod }
+                    };
+                    _hub.AddBreadcrumb(string.Empty, "http", "http", breadcrumbData);
+                }
+
```

Comment:
`Uri.AbsoluteUri` Throws if it's not a AbsoluteUri:

https://github.com/dotnet/runtime/blob/65b472a0f31f2b878601b41a344754ec3e776d1d/src/libraries/System.Private.Uri/src/System/Uri.cs#L707

Also, `request.RequestUri` is already being red above so why `?.` here? The IDE should be warning this is redundant.

Finally, we're using `request.RequestUri` above when adding the span but `request.RequestUri.AbsoluteUri` here, why the inconsistency?

Should we be using only `request.RequestUri` here too?